### PR TITLE
Cyan

### PR DESCRIPTION
**Pull request by scopatz**
Added a recipe for cyan.  CC @rwcarlsen @gonuke @FlanFlanagan @bam241

### CODE REVIEW

**Comment by scopatz** [comment]
Note that this seems to be a legitimate bug in the linter:

```console
scopatz@artemis ~/staged-recipes/recipes/cyan cyan $ conda smithy recipe-lint
/home/scopatz/miniconda/lib/python3.6/site-packages/ruamel/yaml/constructor.py:262: DuplicateKeyFutureWarning: while constructing a mapping
  in "<unicode string>", line 10, column 3:
      fn: cyan  # [unix]
      ^ (line: 10)
found duplicate key "fn" with value "cyan.exe" (original value: "cyan")
  in "<unicode string>", line 11, column 3:
      fn: cyan.exe  # [win]
      ^ (line: 11)

To suppress this check see:
    http://yaml.readthedocs.io/en/latest/api.html#duplicate-keys

Duplicate keys will become and error in future releases, and are errors
by default when using the new API.
```

Note that those lines have selectors for different platforms.

**Comment by rwcarlsen** [comment]
Looks fine - although I'm still not versed in all the intracacies of conda packaging.  I probably should push up some updated binaries with newer embedded sqlite and such.

**Comment by scopatz** [comment]
Thanks for looking at it @rwcarlsen!

**Comment by scopatz** [comment]
@conda-forge-admin, please lint

**Comment by scopatz** [comment]
Hi @conda-forge/staged-recipes - I finally got this working, I think the linter failing is not the fault of this recipe and it should be merged.

File: recipes/cyan/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+source:
+  fn: cyan  # [unix]
+  fn: cyan.exe  # [win]
+  # Linux
+  url: https://github.com/rwcarlsen/cyan/releases/download/v{{ version }}/cyan-linux-amd64 # [linux]
+  sha256: f343ae616c66cc7d27f42f9b68b25135b74188361dd0764de940ff7e2dd0bf2b  # [linux]
+  # Mac
+  url: https://github.com/rwcarlsen/cyan/releases/download/v{{ version }}/cyan-darwin-10.6-amd64  # [osx]
+  sha256: 5670d6ffa9d161eca66dee81115583998ef64a60a5534ebbcc2daaa04b9c81b4 # [osx]
+  # Windows
```
Comment (ocefpaf) [inline recipes/cyan/meta.yaml]: If you remove the `# Linux`, and `# Windows` comments, that are getting interpreted as pre-selectors, the lint will work.
(I would remove the `# Mac` too b/c the pre-selector already brings that info and I would also remove the `fn:` b/c that is not needed.)

Comment (scopatz) [reply recipes/cyan/meta.yaml]: Oh, hah! Weird. OK.

**Comment by scopatz** [comment]
Thanks @ocefpaf! Now the linter passes.

File: recipes/cyan/meta.yaml (hunk 1 above)
Comment (ocefpaf) [reply recipes/cyan/meta.yaml]: Not weird, just buggy parsing, I figure that they were doing some `if line.lower().startswith(# win)`, so...

**Comment by ocefpaf** [comment]
We already know that the recipes is passing, so let's merge this.

**Comment by scopatz** [comment]
Thanks @ocefpaf!

**Pull request merged**